lib/SILOptimizer/Utils/OwnershipOptUtils.cpp

```diff
@@ -233,14 +235,19 @@ static bool hasValidRAUWOwnership(SILValue oldValue, SILValue newValue) {
 // recursiveReborrows.
 static bool canFixUpOwnershipForRAUW(SILValue oldValue, SILValue newValue,
                                      OwnershipFixupContext &context) {
-  if (!hasValidRAUWOwnership(oldValue, newValue))
+  if (!hasValidRAUWOwnership(oldValue, newValue)) {
+    context.clear();
```

Comment:
I don't think we need to clear the context on this one. Can you remove? I don't think this will hurt anything so I am fine with doing it in a forthcoming commit.